library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -724,7 +728,8 @@ public void setOnFeatureClickListener(@NonNull OnFeatureClickListener onFeatureC
     public void focusOnUserLocation(boolean focusOnMyLocation) {
         if (focusOnMyLocation) {
             isMapScrolled = false;
-            changeTargetIcon(R.drawable.ic_cross_hair_blue);
+            boolean isTablet = getResources().getBoolean(R.bool.isTablet);
+            changeImageButtonResource(currentLocationBtn, isTablet ? R.drawable.ic_cross_hair_blue_tablet : R.drawable.ic_cross_hair_blue);
```

Comment:
Is it possible to use the drawables folder for this instead of having this in code?

Is it possible to use drawables configurable for different screen sizes eg. `drawable-sw600dp` and so forth for each mdpi instead of having this in code since that means that you have not covered other DPIs? 

I think it's even easier to manage than having this in code. Yes? No?

Reply:
Fixed.